src/main/java/com/wojciechbarwinski/demo/legendary_warehouse/exceptions/InsufficientStockException.java

```diff
@@ -0,0 +1,18 @@
+package com.wojciechbarwinski.demo.legendary_warehouse.exceptions;
+
+import com.wojciechbarwinski.demo.legendary_warehouse.dtos.InsufficientStockDTO;
+import lombok.Getter;
+
+import java.util.List;
+
+@Getter
+public class InsufficientStockException extends ApplicationException {
+    private final List<InsufficientStockDTO> insufficientStock;
```

Comment:
Jak mamy listę (mnogą liczbę elementów) dobrze jest to odzwierciedlić w nazwie pola/zmiennej

Reply:
racja, ale tu z chęcią bym pogadał bo jakoś ogólnie mam problemy z nazywaniem list xd